docs/GUIDE.md

```diff
@@ -0,0 +1,137 @@
+# VisionPro Meets the World: Video Chat with Agora's Video SDK
+
+ ![](/docs/images/blog_banner.webp "banner")
+
+The future is here, and you're holding it – the Apple VisionPro device. It's a marvel of technology, but sometimes, you just need to see a friendly face across the digital divide. That's where Agora's ubiquitous Real-Time Communication (RTC) SDK comes in. With Agora, video chat on your VisionPro device transcends physical boundaries. You can connect with other VisionPro users, and Agora's expansive reach allows you to chat seamlessly with people on entirely different platforms. Imagine video calling your friends and family on their smartphones, tablets, or laptops – all through the power of Agora on your VisionPro. This quick start guide will equip you to unlock the potential of Agora's native RTC SDK and transform your VisionPro into a powerful video chat hub. 
+
+> If you are a Unity developer, you may refer to [this blog](https://www.agora.io/en/blog/vision-pro-unity-quickstart-with-agora-sdk/) for the corresponding quick start guide.
+
+
+## Prerequisites:
+
+-   Apple VisionPro    
+-   Apple computer with Silicon CPU Chip (M1 or newer generation)
+-   Xcode (15.4)  with VisionOS Support
+-   Agora Developer account
+    
+## Setups
+
+1.  Clone the GitHub project from [this location](https://github.com/AgoraIO-Community/visionOS-Quickstart).
+2. Open the project with Xcode.
+3. Import the Agora SDK plugin. The project includes the Agora SDK as a dependency package. As soon as Xcode loads the project, the package download should start automatically and finish in a few minutes.
```

Comment:
This step is a bit hard to follow. Consider adding a `:` instead of a period after the first statement and combing the following two sentences.

> Import the Agora SDK plugin: The project includes the Agora SDK as a dependency package and the package download should start automatically and finish in a few minutes.